Makefile

```diff
@@ -55,7 +55,12 @@ ifeq ($(ENABLE_FIPS), 1)
 	BUILD_TOOL_CGO_ENABLED := 1
 endif
 
-LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(shell git describe --tags --dirty --always)"
+RELEASE_VERSION ?= $(shell git describe --tags --dirty --always)
+ifeq ($(RUN_CI), 1)
+	RELEASE_VERSION := None
+endif
+
+LDFLAGS += -X "$(PD_PKG)/pkg/versioninfo.PDReleaseVersion=$(RELEASE_VERSION)"
```

Comment:
there is no tag in the ci environment, we can not pass https://github.com/tikv/pd/blob/bfc988bb37a738b3bba2e85903d304af5d8532d6/server/util.go#L39-L41  whithout tag :(